File: Service/Implement/ATTPInfoProductGoodsService.cs
namespace Service.Implement
{
    public class ATTPInfoProductGoodsService : BaseService<ATTPInfoProductGoods, IATTPInfoProductGoodsRepository>
    , IATTPInfoProductGoodsService
    {
    private readonly IATTPInfoProductGoodsRepository _ATTPInfoProductGoodsRepository;
    public ATTPInfoProductGoodsService(IATTPInfoProductGoodsRepository ATTPInfoProductGoodsRepository) : base(ATTPInfoProductGoodsRepository)
    {
    _ATTPInfoProductGoodsRepository = ATTPInfoProductGoodsRepository;
    }
    }
    }

File: CMS/src/app/shared/ATTPInfoProductGoods.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ATTPInfoProductGoods } from 'src/app/shared/ATTPInfoProductGoods.model';
import { BaseService } from './Base.service';
@Injectable({
  providedIn: 'root'
})
export class ATTPInfoProductGoodsService extends BaseService {
  constructor(public httpClient: HttpClient) {
    super(httpClient);
    this.Controller = "ATTPInfoProductGoods";
  }
}

